src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -57,6 +57,15 @@ internal TagHelperExecutionContext(string tagName)
         }
 
         /// <summary>
+        /// Indicates whether or not the tag is self closing.
+        /// </summary>
+        public bool SelfClosing
```

Comment:
1. thought we agreed to name this `OriginalSelfClosing` or `WasOriginallySelfClosing`?
2. "the tag" is a bit ambiguous.  probably want to use "the start tag of the HTML element this context is for". ("the HTML element this context is for" appears elsewhere in this class.)  bit ungainly but at least clear.
3. need to describe original-ness of this value.  definitely include the word "original" in the `<summary/>`.  `<remarks/>` that mention `Output.SelfClosing` reflects changes `ITagHelper` implementations have made may also help.
4. don't think this should be settable since it describes the original start tag.  pass a value in to the `TagHelperExecutionContext` constructor.